components/MyLayout.js

```diff
@@ -67,16 +66,10 @@ class MyLayout extends Component {
   }
 
   render () {
-    const { auth, children } = this.props
+    const { children } = this.props
     const { adminUser } = this.state
-    const { isAuthenticated, user } = auth
 
-    let contents
-    if (isAuthenticated && user && !user.email_verified) {
-      contents = <VerifyEmailScreen />
-    } else {
-      contents = renderChildrenWithProps(children, {...this.state, createUser: this.createUser}) // TODO: find a better way to pass props to children.
-    }
+    const contents = renderChildrenWithProps(children, {...this.state, createUser: this.createUser}) // TODO: find a better way to pass props to children.
```

Comment:
@binh-dam-ibigroup, could you move the TODO onto the line above? Generally, I think comments tend to get lost when they are to the right of code.

Reply:
Fixed